samples/sample_common/src/components/camera_control.cpp

```diff
@@ -47,50 +47,68 @@ namespace nau::sample
         using namespace nau::input;
 
         ImGui::Begin("Component Window");
-        ImGui::SetWindowSize(ImVec2(200, 100), ImGuiCond_Once);
-
+        ImGui::SetWindowSize(ImVec2(300, 100));
+        ImGui::IsWindowHovered();
         ImGui::Text("Component name: %s", reinterpret_cast<const char*>(m_name.c_str()));
 
         ImGui::End();
 
+        // Camera controller settngs 
+        ImGui::Begin("Camera Controll");
```

Comment:
```suggestion
        ImGui::Begin("Camera Control");
```
Suggesting to fix up spelling